### PR TITLE
Fixes #3569 Merged promoted_story_list.xml files 

### LINKED ISSUE #3569: Merge promoted_story_list.xml into single xml file
Currently there are 4 versions of `promoted_story_list.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by mukulasija**
Fixes #3569   Merged 4 promoted_story_list.xml files into single xml file 

mpo

mpn

mlo

mln

ppo

ppn

plo

pln

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by mukulasija** [comment]
@rishidyno PTAL

**Comment by rishidyno** [comment]
LGTM @mukulasija.

**Comment by rt4914** [comment]
> LGTM @mukulasija.

Thanks @rishidyno for reviewing this.

File: app/src/main/res/layout-land/promoted_story_list.xml [hunk 1]
```diff
@@ -1,69 +0,0 @@
...
-
-    <LinearLayout
-      android:layout_width="match_parent"
-      android:layout_height="wrap_content"
-      android:orientation="horizontal"
-      android:gravity="center_vertical">
-
-      <TextView
-        android:id="@+id/recently_played_stories_text_view"
-        style="@style/TextViewStart"
```
Comment (rt4914) [inline app/src/main/res/layout-land/promoted_story_list.xml:28]: This is available only in landscape file. I think this should be used for all files so mention it in mobile-portrait xml file. Also once done please recreate the screenshots.

File: app/src/main/res/layout-land/promoted_story_list.xml [hunk 2]
```diff
@@ -1,69 +0,0 @@
...
-        android:layout_marginStart="72dp"
-        android:layout_weight="1"
-        android:fontFamily="sans-serif-medium"
-        android:text="@{viewModel.getHeader()}"
-        android:textColor="@color/oppia_primary_text"
-        android:textSize="18sp" />
-
-      <TextView
-        android:id="@+id/view_all_text_view"
-        style="@style/TextViewCenterVertical"
```
Comment (rt4914) [inline app/src/main/res/layout-land/promoted_story_list.xml:40]: Ditto

File: app/src/main/res/values-sw600dp-port/dimens.xml [hunk 3]
```diff
@@ -398,6 +398,10 @@
   <dimen name="section_title_text_view_layout_margin_top">32dp</dimen>
   <dimen name="section_title_text_view_layout_margin_end">120dp</dimen>
 
+  <!--  PromotedStoryList-->
+  <dimen name="promoted_story_list_layout_margin_end">120dp</dimen>
+  <dimen name="promoted_story_list_layout_margin_start">120dp</dimen>
```
Comment (rt4914) [inline app/src/main/res/values-sw600dp-port/dimens.xml:403]: `home_tablet_port_outer_margin ` should get removed if not getting used anywhere else.

Comment (rt4914) [reply app/src/main/res/values-sw600dp-port/dimens.xml:403]: Ditto for `home_tablet_land_outer_margin`

**Review by rt4914 — CHANGES_REQUESTED** [review]
@mukulasija Thanks. PTAL at comments.

File: app/src/main/res/values-sw600dp-port/dimens.xml (hunk 3 above)
Comment (mukulasija) [reply app/src/main/res/values-sw600dp-port/dimens.xml:403]: I already removed home_tablet_port_outer_margin and home_tablet_land_outer_margin from dimens file.... and it also doesn't exist in values-sw600dp-port/dimens , can u please tell me which file are u talking about..

File: app/src/main/res/layout-land/promoted_story_list.xml (hunk 1 above)
Comment (mukulasija) [reply app/src/main/res/layout-land/promoted_story_list.xml:28]: I've made the suggested changes and updated the screenshots.  @rt4914 PTAL

**Comment by Arjupta** [comment]
@mukulasija always remeber to ask for a review if you have replied with a question to any comments so that the reviewer can answer them.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**